qiita_plugins/target_gene_type/scripts/start_target_gene_types

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+import click
+
+from target_gene_type.plugin import execute_job
+
+
+@click.command()
+@click.argument('url', required=True)
+@click.argument('job_id', required=True)
+@click.argument('output_dir', required=True)
```

Comment:
Would it make sense to add some *refinement* to the type of argument here, I'm specifically thinking about http://click.pocoo.org/5/arguments/#file-path-arguments